src/stable/f5-bigip-ctlr/templates/f5-bigip-ctlr-clusterrole.yaml

```diff
@@ -0,0 +1,39 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: f5-bigip-ctlr-clusterrole
+  labels:
+    app: {{ template "f5-bigip-ctlr.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - ""
+  - "extensions"
+  resources:
+  - nodes
+  - services
+  - endpoints
+  - namespaces
+  - ingresses
+  - secrets
```

Comment:
Up for debate - consider naming a specific secret instead of granting access to all Kubernetes secrets cluster-wide. I've made this update in the solution doc examples (see https://github.com/F5Networks/f5-ci-docs/pull/401/commits/70798ca108212dcf45c170a3ec8a6a999fa1edcd)

Reply:
I'll need to do some testing to ensure that this change, so not including in this revision.